2021-07-27-minisummit-agenda.md

```diff
@@ -0,0 +1,127 @@
+| 19:00 -- 19:45 | "Qubes OS Native App Menu: UX Design and Implementation" by Marta Marczykowska-Górecka and Nina Alter
+| 19:45 -- 20:15 | Break
+| 20:15 -- 21:00 | "A brief history of USB camera support in Qubes OS" by Piotr Król
+| 21:00 -- 21:45 | "How to setup BTC and XMR cold storage in Qubes OS" by Piotr Król
+| 21:45          | vPub, informal afterparty
+
+## Abstracts of the talks
+
+### "Qubes OS 4.1 highlights" by Marek Marczykowski-Górecki
+
+The upcoming Qubes OS 4.1 release is full of new exciting features, ranging
+from a technology preview of the GUI domain to subtle, yet important, Qrexec
+improvements. In this talk I will give a brief overview of them and demo a
+select few.
+
+### "First Impressions Count: Onboarding Qubes Users Through an Integrated Tutorial" by deeplow
+
+We may all relate to having a rough time when starting using Qubes --- be that
+because we're coming from Windows and everything is different or because we
+come from Linux and many things don't work like we expect them to. Apart from
+the usual challenges of going into a different system, Qubes has the additional
+one of requiring a fundamentally different way of thinking about your computer
+(a hypervisor mental-model). Smoothing out this transition is particularly
+important as Qubes aims to target vulnerable populations that are less
+technically inclined and have less time to explore and read the documentation.
+
+The solution proposed by deeplow is to implement an integrated onboarding
+tutorial. The idea is that a short tutorial (with optional extra parts) that
+guides the user through the essential mechanics of Qubes will make the
+transition simpler. That's what deeplow's been working on for his master's
+dissertation. In this talk he'll introduce the idea and give an update on the
+current progress and challenges.
+
+### "Wyng-backups: revertible local and remote known safe Qubes OS states (including dom0)" by Thierry Laurion
+
+[Wyng-backups](https://github.com/tasket/wyng-backup) is an incremental
+backup/restore tool for LVMs. For Qubes OS, this means even dom0 can be
+reverted to a known safe state; locally or remotely, applying changes only.
+This talk will be a deep dive into the possibilities of wyng-backups for
+deploying and maintaining up to date, revertible states of Qubes OS base
+systems.
+
+### "SRTM and Secure Boot for VMs" by Piotr Król
+
+This talk is the continuation of the Qubes OS mini-summit presentation "SRTM
+for Qubes OS VMs", where the theoretical background of the Static Root of Trust
+was presented and discussed. In this presentation, we will practically approach
+SRTM and Secure Boot for VMs. We will also explore potential use cases for
+self-decrypting storage and signed kernels using safeboot. Finally, we will
+discuss how to introduce this and other security mechanisms in Qubes OS.
+
+### "Usability Within A Reasonably Secure, Multi-Environment System" by Nina Alter
+
+@TODO
```

Comment:
This was a duplicate of the prior entry.

Reply:
fixed